streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java

```diff
@@ -436,9 +436,25 @@ public String getStoreForChangelogTopic(final String topicName) {
         return sourceTopics;
     }
 
-    public Map<Subtopology, TopicsInfo> topicGroups() {
+    /**
+     * @param topologiesToExclude the names of any topologies to exclude from the returned topic groups,
+     *                            eg because they have missing source topics and can't be processed yet
+     */
+    public Map<Subtopology, TopicsInfo> topicGroups(final Set<String> topologiesToExclude) {
```

Comment:
nit: can we update the name to be more descriptive?

Reply:
This method is used a pretty large number of times and I don't want to bloat the PR with style changes, but I totally agree that this name is confusing. I used to never remember what it was and would have to click through to the method implementation and skim the code to figure out what it was doing anytime it came up in the code. Anyways I can file a followup ticket for this

Reply:
Filed https://issues.apache.org/jira/browse/KAFKA-13590